task1.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <link href="task1.css" rel="stylesheet">
+    <meta charset="utf-8">
+    <title>Flowers</title>
+</head>
+<body>
+  <p>
+        <div>
+          <h1>Flowers</h1>
+        </div>
+  <div class="date">
+      <p>November 24th,2018</p>
+  </div>
+  <div class="first">
+      <p><img src="email.png.png" alt="email.png.png" width="40">
+         <img src="user.png.png" alt="user.png.png" width="40">
+         <img src="list.png.png" alt="list.png.png" width="40">
+      </p>
+  <div>
+      <p><img src="phone-book,png.png" alt="phone-book,png.png" width="40">
+         <img src="upload.png.png" alt="upload.png.png" width="40">
+         <img src="placeholder.png.png" alt="placeholder.png.png" width="40">
+      </p>
+  </div>
+  </div>
+       
+  </p>
+       <h2>Разнообразие видов цветов</h2>
+    <div class="place">
+       <h3>Самый древний цветок</h3>
+       <p><img src="flowers4.png" alt="flowers4.png">
+       Лотос - один из самых древних цветов на земле — существовали они уже в меловом периоде, больше 100 млн. лет назад.
+       Их ископаемые останки находят и в Северной Америке, и на Дальнем Востоке, и даже в Арктике. 
+       В наше время насчитывают, строго говоря, всего два вида чистокровных лотосов.
+       Самый древний цветок, «мать всех цветов», был обнаружен археологами в 2002 г. на территории современного Китая. 
+       Его возраст более 125 000 000 лет.
+       </p>
+    </div>
+    <div class="part">
+        <h4>Воскрешающий лотос</h4>
+        <p><img src="flowers7.png" alt="flowers7.png">
+            В Древнем Египте лотос считали священным цветком, символом воскрешения. 
+            Ему поклонялись и применяли в погребальных обрядах. 
+            Это во многом обусловлено способностью лотоса выживать во время многолетней засухи и начинать цвести лишь почва снова станет влажной.
+          </p>
+    </div>
+    <div class="long">
+        <h5>Солнцелюбивые подсолнухи</h5>
+        <p><img src="flowers5.png" alt="flowers5.png">

```

Comment:
Данный код ничего не скажет ни разработчику, ни пользователю. Best practice:
```suggestion
        <p><img src="sunflower.png" alt="sunflower">
```